packages/components/organisms/f-checkout/src/store/tests/checkoutAnalytics.module.test.js

```diff
@@ -380,6 +381,31 @@ describe('CheckoutAnalyticsModule', () => {
                 expect(dispatch).toHaveBeenCalledWith('trackFormInteraction', { action: 'error', error: errors });
             });
         });
+
+        describe('trackDuplicateOrderWarnDialog ::', () => {
+            it('should `push` expected event to `dataLayer`', () => {
+                // Arrange
+                const expected = {
+                    event: 'trackEvent',
+                    eventCategory: 'engagement',
+                    eventAction: 'dialog_duplicate_order_warning',
+                    eventLabel: 'view_dialog'
+                };
+
+                Object.defineProperty(global, 'window', {
```

Comment:
This is already defined here, so no need to do it again https://github.com/justeat/fozzie-components/pull/1003/files/e9c58d7a9e9e97cfddacfd6a845f2a3d2abcea7d..39bfe636e4b4c370e0873b41591f877f44360e9e#diff-0110d53ae1c387a8c57603e99682e6047f873042c29e6ecaf3646722a8da5e08R22